tool/speed.cc

```diff
@@ -2409,10 +2408,8 @@ static bool SpeedPKCS8(const std::string &selected) {
 
   CBS in;
 
-  CBS_init(&in, CBB_data(&out), CBB_len(&out));
-
-
-  if (!TimeFunction(&results, [&in]() -> bool {
+  if (!TimeFunction(&results, [&in, &out]() -> bool {
+        CBS_init(&in, CBB_data(&out), CBB_len(&out));
```

Comment:
Why was this call brought into the loop benchmark? would it be intended to be called with the parsing function every time?

Reply:
If it's not in the loop the data in the CBS is consumed and if EVP_parse_private_key runs more times than copies of the key exist in the CBS the benchmark fails. Init is very fast and just stores a reference to the data and length, but it does slightly change the measured time.